usaspending_api/search/v2/views/spending_by_category_views/spending_by_locations.py

```diff
@@ -105,3 +109,36 @@ class CountyViewSet(AbstractLocationViewSet):
 
     location_type = LocationType.COUNTY
     category = Category(name="county", agg_key="pop_county_agg_key")
+
+
+class DistrictViewSet(AbstractLocationViewSet):
+    """
+    This route takes award filters and returns spending by Congressional District.
+    """
+
+    endpoint_doc = "usaspending_api/api_contracts/contracts/v2/search/spending_by_category/district.md"
+
+    location_type = LocationType.CONGRESSIONAL_DISTRICT
+    category = Category(name="district", agg_key="pop_congressional_agg_key")
```

Comment:
Slightly confused why both `location_type` and `name` are necessary and set here.

Reply:
Continuing forward current functionality, the `congressional_district` category is referred to as `district` but the database stores these values as `(pop | rescipient_location)_congressional_code`. Same with state where the category is `state_territory`, but the database has columns as `(pop_recipient_location)_state_code`. Have them specified differently allowed for code to be a little bit cleaner since we can keep the term expected by database and term expected as the "category" separate. Ideally, these would be the same and we would be consistent across the board.

Reply:
I understand the field name discrepancies (confusion).

Not a big enough deal to hold up this PR, perhaps a future refactor can reuse the `location_type`  for Category and remap to the desired values